content/cybersecurity/concepts/blue-team/blue-team.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Blue Team'
+Description: 'Blue team typically consists of incident response professionals who try to stop sophisticated attacks.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Ethical Hacking'
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/scan-systems-with-nmap'
+  - 'introduction-to-ethical-hacking'
+---
+
+**Blue team** typically consists of incident response professionals who try to stop sophisticated attacks. The blue team's goal is to protect the organization's critical assets or crown jewels.
+
+## Blue Team In-Use:
+
+Some of the tasks of a blue team member might include digital footprint analysis, installing and configuring firewalls, and monitoring network activity. Blue teams often start by conducting a detailed risk assessment of the organization’s current security program to identify potential threats and weaknesses. The blue team will help educate employees on best practices in cybersecurity procedures, and often stronger password policies are implemented to tighten access to the system.
+
+## Blue Team vs. Red Team: 
+
+Blue teams conduct risk assessments and provide relevant mitigation tools for companies to better gauge their defenses. An organization may schedule red team vs. blue team exercises in which red teams may attempt various techniques to launch an attack. The blue team is tasked with trying to repel these attacks and expose the red team activity. Ultimately, the blue team must prevent any data breaches and then remediate any uncovered vulnerabilities.
+
+Conversely, a red team will use a variety of tactics such as social engineering, penetration testing, and physical security breaches to emulate the methods that an attacker might use. A red team will probe for weaknesses that traditional security measures might not detect. After an attack simulation, a red team can provide an after action report outlining any vulnerabilities found and offering ways to remediate them.
+
+## Skills Needed for a Blue Team Member:
+
+The blue team’s job is to prevent and detect attacks. Common skills for the blue team include, but are not limited to:
+
+- Detailed knowledge of concepts such as firewalls, intrusion detection systems and antivirus software
```

Comment:
```suggestion
Detailed knowledge of concepts such as firewalls, intrusion detection systems, and antivirus software.
```

Reply:
Actually, don't remove the leading dash as this is a bullet point.